aggsender/aggsender.go

```diff
@@ -131,9 +142,20 @@ func (a *AggSender) Start(ctx context.Context) {
 	a.log.Info("AggSender started")
 	metrics.Register()
 	a.status.Start(time.Now().UTC())
+	a.checkDBCompatibility(ctx)
 	a.checkInitialStatus(ctx)
 	a.sendCertificates(ctx, 0)
 }
+func (a *AggSender) checkDBCompatibility(ctx context.Context) {
+	if a.compatibilityStoragedChecker == nil {
+		a.log.Warnf("compatibilityStoragedChecker is nil, so we are not going to check the compatibility")
+		return
+	}
+	err := a.compatibilityStoragedChecker.Check(ctx, nil)
+	if err != nil {
+		a.log.Fatalf("error checking compatibility data in DB, you can bypass this check using config file")
```

Comment:
Incorporate the error message to the log as well?
```suggestion
		a.log.Fatalf("error checking compatibility data in DB, you can disable this check using config file: %s", err)
```

Reply:
About adding the Error ok!, about changing 'bypass' to 'disable' I think  that `bypass` is more accurated, beucase you execute the test, but just ignore the error. WDTY?

Reply:
Ok, leave it as a bypass, but consider adding an error to the log.